### DISCUSSION TITLE
New to Deepzoom

### DISCUSSION THREAD

**Discussion by kayaker123** [jaybo/deepzoom-blog#20] [General]
Hi This is a very basic question. Last week I created my first trip in Deepzoom and saved it as a private trip. Today I logged into Deepzoom and can't find it again. I found all the public trips in the area, but my trip is no where to be found. Can you help me with finding it again. Thank you

**Comment by jaybo** [comment]
If you're seeing the demo trips, I'm pretty sure you're not logged in.  Note that it's possible to be logged-in on one browser tab, but not logged-in on another tab.

**If logged in:**

![image](https://user-images.githubusercontent.com/1607323/229927340-42e652d0-031b-4e0f-9865-5ed14a889e64.png)

**If not logged in:**

![image](https://user-images.githubusercontent.com/1607323/229926747-28097cba-6e04-4a21-849f-588c771edc8c.png)
